int64 mode
float64[5] para
---
string check
